src/main/java/javax/validation/Path.java

```diff
@@ -198,6 +200,20 @@
 	 * @since 1.1
 	 */
 	interface BeanNode extends Node {
+
+		/**
+		 * @return the class of the container for the element affected by the violated constraint
```

Comment:
the tpye of the container the node is placed in if contained in a container type such as `Optional`, ` List` or an array {@code null} otherwise